android-components/components/support/utils/src/test/java/mozilla/components/support/utils/WebURLFinderTest.kt

```diff
@@ -100,12 +100,20 @@ class WebURLFinderTest {
     }
 
     @Test
-    fun testIsWebURL() {
-        assertTrue(isWebURL("http://test.com"))
-        assertFalse(isWebURL("#http#://test.com"))
-        assertFalse(isWebURL("file:///sdcard/download"))
-        assertFalse(isWebURL("filE:///sdcard/Download"))
-        assertFalse(isWebURL("javascript:alert('Hi')"))
-        assertFalse(isWebURL("JAVascript:alert('Hi')"))
+    fun testIsValidWebURL() {
```

Comment:
should we add an assertion for `content:`?

Reply:
We absolutely should, I thought I had tbh Good spot!